src/pages/docs/infrastructure/workers/autoscaling-workers/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-08-22
+modDate: 2024-08-22
+title: Auto Scaling
```

Comment:
This page is written in the context of the k8s worker but the title isn't, I'd suggest changing this to:

> Auto Scaling with the Kubernetes Worker